### DISCUSSION TITLE
Border Problemi

### DISCUSSION THREAD

**Discussion by AliEkremDinkcioglu** [twbs/bootstrap#37993] [General]
Merhaba, 5.3.1 den sonraki sürümlerde border-1 , -2, -3, -4, -5 sınıfları calışmıyor. Sorun sadece bende mi?
![image](https://user-images.githubusercontent.com/43677930/216341035-83de5a83-825a-401e-adaa-1cc6998fa227.png)
![image](https://user-images.githubusercontent.com/43677930/216341106-0642de5d-06ec-4799-adcd-615c02b37869.png)

olması gereken
![image](https://user-images.githubusercontent.com/43677930/216341545-17055f80-16b5-43d6-8b38-fbc9d0f75ac3.png)

**Comment by denizgolbas** [comment]
https://getbootstrap.com/docs/5.3/utilities/borders/#border

It works for [5.3.0-alpha1](https://blog.getbootstrap.com/2022/12/24/bootstrap-5-3-0-alpha1/)
https://jsfiddle.net/b9ruokvp/2/
